Apps/BRNPlayground/README.md

```diff
@@ -0,0 +1,12 @@
+# BabylonReactNative Playground App
+
+This is a new example app for BabylonReactNative, based on `react-native-test-app`.
```

Comment:
Neither this app nor the original Playground app are actually a "Playground." Ideally this app would be called something more like TestApp, indicating it's purpose is to use all/many BRN/BN features so we can easily test them (manual or future UI automation). Not sure how involved renaming it would be though.